src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2774,6 +2774,11 @@ void ApplicationManagerImpl::PullLanguagesInfo(const SmartObject& app_data,
     SDL_LOG_DEBUG("Get ttsName from " << cur_vr_lang << " language");
     ttsName =
         app_data[json::languages][specific_idx][cur_vr_lang][json::ttsName];
+  } else if (app_data[json::languages][default_idx][json::default_].keyExists(
+                 json::ttsName)) {
+    SDL_LOG_DEBUG("Get ttsName from default language");
+    ttsName =
+        app_data[json::languages][default_idx][json::default_][json::ttsName];
```

Comment:
@Ypostolov Please notice `ttsName` is an array according to the https://github.com/smartdevicelink/sdl_core/tree/develop/src/components/interfaces/HMI_API.xml#L2914.
However within fix we got non-array value, e.g.
```
"ttsName": "A",
```
In Json file it's defined as non-array, but it looks like we need to convert it to array with single element.

Reply:
@dboltovskyi  
Fixed in ce8706ba5e3dfb67b89975dd0997ca44a744ac23. 
Please check regression.

Reply:
@dboltovskyi  
Fixed in ce8706ba5e3dfb67b89975dd0997ca44a744ac23.
Please check regression.